test/functional/rpc_getgeneralinfo.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the getgeneralinfo RPC."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, try_rpc
+
+class GetGeneralInfoTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = False
+        self.num_nodes = 1
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_cli()
+
+    def run_test(self):
+        """Test getgeneralinfo."""
```

Comment:
Please use `#` if it's a one line comment

Reply:
Please, check previous reviews from @jonatack 
I have already changed those comments from # to doctype.

Reply:
I agree with @emilengler and would suggest spending some time looking at the other tests and the PEP8 style guide.